react-app/src/containers/RequestPage/RequestPage.css

```diff
@@ -0,0 +1,8 @@
+.Dropdown {
+    display: grid !important;
```

Comment:
It is generally a best practice to avoid using !important in CSS code. Is this really needed here? Grid displays should work without needing !important.